src/devices/machine/intelfsh.h

```diff
@@ -347,63 +341,83 @@ class atmel_49f4096_device : public intelfsh16_device
 	atmel_49f4096_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock = 0);
 };
 
-class cat28f020_device : public intelfsh8_device
+class tc58fvt800_device : public intelfsh16_device
```

Comment:
intelfsh8_device and intelfsh16_device devices are separated in the header but intelfsh16_device devices were mixed in with the intelfsh8_device devices so I moved them into the intelfsh16_device group for consistency.